healthlocker.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description="example app_name daemon"
+
+[Service]
+Type=simple
+User=root
+Restart=on-failure
+Environment=MIX_ENV=prod "PORT=4000"
+Environment=LANG=en_US.UTF-8
```

Comment:
is the Language `en_US` ?

Reply:
Thanks for spotting this. I will look into this var and see why is was recommended that I include it. 👍

Reply:
After doing a little reading this doesn't look like it is needed at all and it is just something that I left in in error from an example I saw. Thanks @nelsonic